v2/pkg/protocols/common/generators/generators.go

```diff
@@ -2,49 +2,53 @@
 
 package generators
 
-import "github.com/pkg/errors"
+import (
+	"github.com/pkg/errors"
+	"github.com/projectdiscovery/nuclei/v2/pkg/catalog"
+)
 
 // Generator is the generator struct for generating payloads
 type Generator struct {
```

Comment:
either the `package` name or the `struct` should contain the word `payload`, otherwise you don't know what kind of generator you are working with..